site/source/tutorials/1010_containers_and_injecting_state/index.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: tutorial
 title: State management
-overview: In this tutorial, you will learn what Containers and Injectors are and how to use them to coordinate external state and inject as properties into the sections of the widget tree.
+overview: Learn what Containers and Injectors are.  Use them to manage your applications state and provide that state to widgets.
```

Comment:
They don't really manage applications state, more they do in fact help coordinate or side load external state (managed by something else, aka stores) and map and inject this state into a widget via properties.

Reply:
Tweaked it...